src/pages/blog/video-banner/_VideoCarousel.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import VideoSrc from '../../markets/static/video/globe.mp4'
+import VideoPlayer from './_video-player'
+import { CustomCarousel } from './carousel/_custom-carousel'
+import { Flex } from 'components/containers'
+import { Header } from 'components/elements'
+import { LinkButton } from 'components/form'
+import device from 'themes/device'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+import PlayIcon from 'images/svg/blog/video/Triangle.svg'
```

Comment:
We gotta remember to use the same one once merged with my PR.

Reply:
Noted on this..also on color variables.